specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -249,7 +256,12 @@ function RecordSet<SCHEMA extends AnySchema>({
   // Fetch ID of record at current index
   const currentRecordId = ids[currentIndex];
   React.useEffect(() => {
-    if (currentRecordId === undefined) handleFetch(currentIndex);
+    if (currentRecordId === undefined) {
+      if (recordSet.isNew()) {
+        setIds([currentRecord.id]);
```

Comment:
redundant `{` and `}`. There is no need to add them when you have just a single statement.

Given how easy it is to confuse `}` and add extra ones or remove some accidentally, keeping the code clean and avoiding `{` whenever possible is advisable